docs/source/asr/asr_language_modeling.rst

```diff
@@ -69,27 +70,39 @@ sample. For JSON manifest file, the file need to contain json formatted samples
-| nemo_model_file  | str      | Required    | The path of the `.nemo` file of the ASR model. It is needed to extract the tokenizer.           |
+| nemo_model_file  | str      | Required    | The path to `.nemo` file of the ASR model, or name of a pretrained NeMo model to extract a tokenizer. |
 +------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
-| train_file       | str      | Required    | Path to the training file, it can be a text file or JSON manifest.                              |
+| train_paths      | List[str] | Required    | List of training files or folders. Files can be a plain text file or ".json" manifest or ".json.gz". |
 +------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
 | kenlm_model_file | str      | Required    | The path to store the KenLM binary model file.                                                  |
 +------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
 | kenlm_bin_path   | str      | Required    | The path to the bin folder of KenLM. It is a folder named `bin` under where KenLM is installed. |
 +------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
 | ngram_length**   | int      | Required    | Specifies order of N-gram LM.                                                                   |
 +------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
-| do_lower_case    | bool     | ``False``   | Whether to make the training text all lower case.                                               |
+| ngram_prune      | List[int] | [0]        | List of digits to prune Ngrum. Example: [0,0,1]. See Pruning section on the https://kheafield.com/code/kenlm/estimation  |
++------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
+| cache_path       | str      | ""          | Cache path to save tokenized files.                                                             |
++------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
+| text_processing.do_lowercase      | bool | ``False`` | Whether to make the training text all lower case.                                    |
++------------------+----------+-------------+-------------------------------------------------------------------------------------------------+
+| text_processing.punctuation_marks | str   | ""       | String with punctuation marks to process. Example: ".,?"                             |
```

Comment:
Should there be a delimiter? how ellipsis is going to be handled?

Reply:
Currently ellipsis and other language dependend text cleaning algorithms separated from this repo. I think to put it to NeMo-text-processing repo.